src/controllers/sharedExecutionPlanUtils.ts

```diff
@@ -107,6 +107,15 @@ export async function createExecutionPlanGraphs(
             // malformed xml
             newState.loadState = ApiStatus.Error;
             newState.errorMessage = getErrorMessage(e);
+            state.executionPlanState = newState;
+
+            sendErrorEvent(
+                TelemetryViews.ExecutionPlan,
+                TelemetryActions.OpenExecutionPlan,
+                e,
+                false,
```

Comment:
probably okay to include the error info since it's just an XML parsing exception